sympy/geometry/polygon.py

```diff
@@ -2270,6 +2270,69 @@ def exradii(self):
         return exradii
 
     @property
+    def excenters(self):
+        """Excenters of the triangle.
+
+        An excenter is a point at which the line bisecting one interior angle meets the
```

Comment:
Although this is an interesting definition of the excenter location, perhaps it would be clearer to describe it relative to the circle that it defines:

An excenter is the center of a circle that is tangent to a side of the triangle and the extensions of the other two sides.